ruby/src/api.cc

```diff
@@ -469,7 +469,7 @@ namespace leatherman { namespace ruby {
                                                                   "break file if File.exist? file;"
                                                                   "false end)" });
         if (!success) {
-            LOG_WARNING("ruby failed to run", output, none);
+            LOG_WARNING("ruby failed to run");
```

Comment:
Comparing te [stable](https://github.com/puppetlabs/facter/blob/stable/lib/src/ruby/api.cc#L509), this should be `"ruby failed to run: %1%", output`.